cmd/arcstat/arcstat

```diff
@@ -335,15 +334,15 @@ def init():
         i += 1
 
     argv = sys.argv[i:]
-    sint = Decimal(argv[0]) if argv else sint
+    sint = int(argv[0]) if argv else sint
     count = int(argv[1]) if len(argv) > 1 else count
 
     if len(argv) > 1:
-        sint = Decimal(argv[0])
+        sint = int(argv[0])
         count = int(argv[1])
 
     elif len(argv) > 0:
-        sint = Decimal(argv[0])
+        sint = int(argv[0])
         count = 0
```

Comment:
We should get rid of these lines entirely and fix the previous line as follows:
```python
		     count = int(argv[1]) if len(argv) > 1 else 0
```

Reply:
Thank you for the suggestion!
The code has been cleaned: https://github.com/openzfs/zfs/pull/10142/commits/eeef30083555efc6cc68501c5e8f4d6617d5ac44
Now it looks much better :)

Reply:
@freqlabs, it looked fine, but the code changed its behavior.
Before the change:
```
$ arcstat 
    time  read  miss  miss%  dmis  dm%  pmis  pm%  mmis  mm%  arcsz     c  
23:17:56     0     0      0     0    0     0    0     0    0   7.8G  7.7G
```
After the change:
```
$ ./arcstat
    time  read  miss  miss%  dmis  dm%  pmis  pm%  mmis  mm%  arcsz     c  
23:18:00     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:01     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:02     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:03     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:04     0     0      0     0    0     0    0     0    0   7.8G  7.7G
```

Sorry for the late testing :/ I will prepare a fix for it.

Reply:
@freqlabs, here is the fix: https://github.com/openzfs/zfs/pull/10142/commits/b51698ffd20aae87807c3ecf11f8bffba138876d
Does it look fine? I added brackets for better readability.

Reply:
Looks good.